application/src/main/java/org/opentripplanner/standalone/config/framework/json/ConfigType.java

```diff
@@ -71,6 +71,13 @@ public enum ConfigType {
   ),
   TIME_ZONE(JsonType.string, "Time-Zone ID", "UTC", "Europe/Paris", "-05:00"),
   FEED_SCOPED_ID(JsonType.string, "FeedScopedId", "NO:1001", "1:101"),
+  GRAM(
```

Comment:
Again, should this not be WEIGHT?

Reply:
@we should keep it consistent with the `Gram` class - whatever we decide.